build.gradle

```diff
@@ -66,8 +66,16 @@ allprojects {
         }
     }
 
+    task updateInstallerVersion(type: WriteProperties) {
+        exec {
+            def installerVersionVal=UUID.randomUUID().toString()
+            commandLine 'sh', '-c', "sed -ie \"s/^installerVersion=.*/installerVersion=$installerVersionVal/\" /Users/parkavi/Documents/Parkavi/Projects/Ballerina-Distribution/Forked/ballerina-distribution/gradle.properties"
```

Comment:
@parkavi11 This is gradle.properties file of your machine. Shall we change this to the relative path?